crates/subspace-verification/src/lib.rs

```diff
@@ -339,6 +340,14 @@ pub fn derive_randomness<PublicKey, RewardAddress>(
     ]))
 }
 
+/// Derive proof of time entropy from solution for injection purposes.
+pub fn derive_pot_entropy<PublicKey, RewardAddress>(
```

Comment:
Entropy should be derived from `chunk` and `proof_of_time` (sorry for the delayed spec update), so I think you would want to pass PreDigest here. Also `slot` is not necessary.